vm-allocator/src/lib.rs

```diff
@@ -19,4 +19,4 @@ mod system;
 
 pub use crate::address::AddressAllocator;
 pub use crate::gsi::{GsiAllocator, GsiApic};
-pub use crate::system::SystemAllocator;
+pub use crate::system::{pagesize, SystemAllocator};
```

Comment:
Let's not do that because the `vm-allocator` is not really the place for declaring so generic syscalls. Instead, you can simply duplicate the code where you need it, and in parallel, submit this syscall to the `vmm-sys-util` crate.

Reply:
I suggested to export it from the arch crate. Not exporting and duplicating is also ok for such a trivial call.

Reply:
Oh ok! Don't you think the `vmm-sys-util` crate should be the right place for the long term?

Reply:
> Oh ok! Don't you think the `vmm-sys-util` crate should be the right place for the long term?

In which part of the `vmm-sys-util` crate would you include it?

Reply:
I don't know about the exact part, but I thought anything related to syscalls should be defined in `vmm-sys-util`.

Reply:
so what's the conclusion about this? put it in `arch/src/lib.rs` for now?

Reply:
> so what's the conclusion about this? put it in `arch/src/lib.rs` for now?

Let's duplicate it for now. I don't want the allocator crate to depend on arch.

Reply:
done